source/Preferences.h

```diff
@@ -135,6 +141,10 @@ class Preferences {
 	static VSync VSyncState();
 	static const std::string &VSyncSetting();
 
+	static void ToggleCameraAcceleration();
+	static const std::string &CameraAccelerationName();
+	static const CameraAccel CameraAcceleration();
```

Comment:
The string getters are generally called "Setting", not "Name".
They're also the last method in their groups.
There's not much point making the method returning a copy of the enum value `const`. It returns a copy. The others also aren't const.
```suggestion
	static void ToggleCameraAcceleration();
	static CameraAccel CameraAcceleration();
	static const std::string &CameraAccelerationSetting();
```